images/hook/entrypoint.sh

```diff
@@ -8,12 +8,6 @@ if [ $1 = "update" ]; then
     echo "---> Creating monitoring namespace"
     /opt/bin/kubectl apply -f /var/lib/gravity/resources/namespace.yaml
 
-    echo "---> Getting node name for influxdb pod"
```

Comment:
That part of the code was always broken because we are draining nodes before starting this upgrade.